asset/js/setup.js

```diff
@@ -95,6 +95,18 @@ function setupSiteNav() {
   );
 }
 
+function setupReadingProgress() {
+  const progress = document.querySelector('.progress');
+
+  document.addEventListener('scroll', () => {
```

Comment:
Oops, one more thing! We should only add the event listener when the progress indicator is on the page. Otherwise, `progress` might not be defined.
<img width="837" alt="Screenshot 2020-01-20 at 5 04 47 PM" src="https://user-images.githubusercontent.com/17447681/72712908-fccb1900-3ba6-11ea-932e-c8c89fe36634.png">

Reply:
Fixed